packages/teams-js/test/internal/communication.spec.ts

```diff
@@ -304,6 +304,370 @@ describe('Testing communication', () => {
       });
     });
   });
+  describe('uninitializeCommunication', () => {
+    let utils: Utils = new Utils();
```

Comment:
I know it's not something you're introducing, but it's really weird to see `Utils` as a class you instantiate. Shouldn't all utility things be stateless?

Reply:
Looks like we all agree!